tools/generators/open-feature/files/shared/babel.config.json__tmpl__

```diff
@@ -1,3 +1,3 @@
 {
-  "presets": ["minify"]
+  "presets": [["minify", { "builtIns": false }]]
```

Comment:
This was a change I made in every lib to workaround a babel bug: https://github.com/babel/minify/issues/904

Including Math functions (`Math.min`) causes a packaging failure.